### DISCUSSION TITLE
Using different type of editor

### DISCUSSION THREAD

**Discussion by mark-15** [payloadcms/payload#1956] [Q&A] [answered]
Hey Guys,

I just wanna ask if there is a possibility that I can use [ckeditor](https://ckeditor.com/) instead of [slatejs](https://www.slatejs.org/)

**Comment by zubricks** [comment, marked as answer]
Hey @mark-15 anything is possible, though we don't officially support anything other than slate. We have a community member that built a [plugin for Lexical,](https://github.com/payloadcms/payload/discussions/1772) but nothing for ckeditor at this time.

**Reply by jmikrut** [reply]
To expand on this, using ckeditor would actually be quite easy. Because it stores its contents as a string, you would just need to make a custom Field component for any given text field.

Then, within that custom field component, you would render ckeditor accordingly. Would be easy!
